pick.c

```diff
@@ -31,23 +31,25 @@
 } while (0)
 
 enum key {
-	UNKNOWN,
```

Comment:
Although, I do agree this list should be sorted. But lets skip that for
now and produce the smallest possible diff. Just add `END` and `HOME`
before `PRINTABLE`.